syntax = "proto3";

package predictionmarkets.auth;

message UsernameLogInRequest {
    string username = 1;
    string password = 2;
}

message UsernameLogInResponse {
    string entity_id = 1;
}

message LogOutRequest {
}

message LogOutResponse {
}
